UnityProject/Assets/Scripts/Core/Editor/EditorDrawPreview.cs

```diff
@@ -102,6 +102,7 @@ static EditorDrawPreview()
 
 	private static void DrawProjectItem(string guid, Rect selectionRect)
 	{
+		if (EditorPrefs.GetBool("editorPreviewsDisable")) return;
```

Comment:
```suggestion
		if (EditorPrefs.GetBool("editorPreviewsDisable"), false) return;
```

don't you need to give it a default value if not present?